JKNS-257: updates around new multi container sample pod templates in the openshift jenkins image (#39610)
openshift/openshift-docs: modules/images-other-jenkins-create-service.adoc

----
$ oc new-app jenkins-ephemeral
----

> With both templates, you can run 'oc describe' on them to see all the parameters available for overriding.

Reviewer: ```suggestion
With both templates, you can run `oc describe` on them to see all the parameters available for overriding.
```

----
$ oc new-app jenkins-ephemeral
----

With both templates, you can run 'oc describe' on them to see all the parameters available for overriding

> various settings for the resulting Jenkins deployment, including several of the xref:images-other-jenkins-env-var.adoc[environment variables] available

Reviewer: ```suggestion
```

> on the image.

Reviewer: ```suggestion
```
